src/api/stats/getMultichainVaults.js

```diff
@@ -81,18 +95,36 @@ function buildFromChains() {
   serviceEventBus.emit('vaults/updated');
 }
 
+function buildFromGovChains() {
+  multichainGovVaults = Object.values(govVaultsByChain).flat();
+  multichainGovVaultsCounter = multichainGovVaults.length;
+  multichainGovActiveVaultsCounter = sumBy(multichainGovVaults, vault =>
+    vault.status === 'active' ? 1 : 0
+  );
+  vaultsByID = keyBy(multichainGovVaults, 'id');
```

Comment:
Careful, you're overwriting `vaultsByID` with the govVaults, this should be done on a separate variable